src/pages/Lending/Pool/components/PoolInfo.tsx

```diff
@@ -107,7 +120,65 @@ export const PoolInfo = ({ poolAssetId }: PoolInfoProps) => {
         {translate('common.halted')}
       </Tag>
     )
-  }, [poolData, translate])
+  }, [isThorchainLendingBorrowEnabled, isThorchainLendingRepayEnabled, poolData, translate])
+
+  const poolAlert = useMemo(() => {
+    if (!poolData) return null
+
+    if (!isThorchainLendingBorrowEnabled && !isThorchainLendingRepayEnabled) {
+      return (
+        <Alert status='warning' variant='subtle'>
+          <AlertIcon />
+          <AlertDescription>
+            {translate('lending.haltedAlert')}
+            <Link isExternal href='https://discord.gg/n7F4z5Cn' ml={1} color='text.link'>
+              {translate('lending.halterMoreDetails')}
+            </Link>
+          </AlertDescription>
+        </Alert>
+      )
+    }
+
+    if (poolData.isHardCapReached || bnOrZero(poolData.currentCapFillPercentage).eq(100)) {
+      return (
+        <Alert status='warning' flexDir='column' bg={alertBg} py={4}>
+          <AlertIcon />
+          <AlertTitle>{translate('defi.modals.saversVaults.haltedDepositTitle')}</AlertTitle>
```

Comment:
preferably-blocking: this is savers-specific and says `Deposits are temporarily halted.` where this should say "Borrows".

Not super concerned about it because of the current state of lending but...

Reply:
Also it's not a diff of the actual PR but an historical thing, left as it is for now, if you really care, we could ask product